openwisp_controller/subnet_division/filters.py

```diff
@@ -0,0 +1,78 @@
+from django.contrib import admin
+from django.db.models import Q
+from swapper import load_model
+
+from . import settings as app_settings
+
+SubnetDivisionIndex = load_model('subnet_division', 'SubnetDivisionIndex')
+Subnet = load_model('openwisp_ipam', 'Subnet')
+
+
+class InputFilter(admin.SimpleListFilter):
+    template = 'admin/input_filter.html'
+
+    def lookups(self, request, model_admin):
+        # Required to show the filter.
+        return [tuple()]
+
+    def choices(self, changelist):
+        # Grab only the "all" option.
+        all_choice = next(super().choices(changelist))
+        all_choice['query_parts'] = []
+        for key, value in changelist.get_filters_params().items():
+            if key != self.parameter_name:
+                all_choice['query_parts'].append((key, value))
+        yield all_choice
+
+
+class SubnetFilter(InputFilter):
+    parameter_name = 'subnet'
+    title = 'Subnet'
```

Comment:
```suggestion
    title = _('subnet')
```

Keep in mind all strings presented to users must be flagged as translatable.
Use non capitalized for consistency.